app/js/app/directives/graph_sketcher/GraphView.js

```diff
@@ -34,14 +34,14 @@ define(['./GraphUtils.js'], function(graphUtils) {
             // want to connect closest points x,y wise, not just x wise
             let pts = curve.pts;
             for (let i = 1; i < pts.length; i++) {
-                if (pts[i].x - pts[i-1].x < 100 && pts[i].y - pts[i-1].y < 100) {
-                    this.p.line(pts[i-1].x, pts[i-1].y, pts[i].x, pts[i].y);
+                if (pts[i][0] - pts[i-1][0] < 100 && pts[i][1] - pts[i-1][1] < 100) {// TODO why <100?
```

Comment:
Another magic number, at least with a TODO this time; replace TODO with explanation.